wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -116,26 +118,28 @@ class WireGenerator(
 }
 
 private fun parseFileDescriptor(fileDescriptor: DescriptorProtos.FileDescriptorProto, descs: DescriptorSource): ProtoFileElement {
-  val path = mutableListOf<Int>()
   val location = Location.get(fileDescriptor.name)
   val helper = SourceCodeHelper(fileDescriptor)
 
   val imports = mutableListOf<String>()
   val publicImports = mutableListOf<String>()
   val types = mutableListOf<TypeElement>()
 
+  val path = mutableListOf<Int>()
   path.clear()
```

Comment:
I guess we don't need `clear()` now that we've moved init down. (I had only put `clear` here, since it's required before re-using the path... so if we ever added a use above this line, I wanted to make sure it was reset before this point.)